crossteam-project-valuation.md

```diff
@@ -0,0 +1,101 @@
+# Title
+
+Cross-team Project Valuation
+
+# Patlet
+
+It's hard to sell the value of cross-team, inner sourced projects that don't provide a direct impact on company revenue.
+Here's a data-driven way to represent your project that both articulates its value and amplifies it.
+
+# Context
+
+* You are responsible for a cross-team that serves as a platform for others in the company.
+* The cross-team project does not deliver any direct value for company revenue.
+
+# Problem
+
+Cross-team projects can potentially have a very large impact on the company yet are notoriosly-difficult to represent in a data-driven fashion.
+As a result, it is easy and common to either pursue projects that does not provide real value or to underfund what would otherwise produce great value.
+Setting in place a pattern and model of how to value such projects gives us the tool that we need to focus and amplify high-value collaboration for the company.
+
+# Forces
+
+Projects need to show value (objective or subjective) to company leadership in order to be funded.
+
+# Solution
+
+The core of all cross-team project value is the idea that we can get more done together than apart.
+Ascribing value to a cross-team effort is an exercise in quantifying _how much more_ is getting done together.
+The exact delta in productivity will vary by domain and project.
+There is a common process, though, by which you can create a model to calculate it.
+
+## Explanation
+
+Assemble a small team of subject matter experts in your domain.
+Using that team of experts, estimate 4 things about each consumer of your project output:
+  * How long does it take them to consume your project output?
+  * How long would it otherwise take them to home-roll the value of your project output for themselves?
+  * What percentage of your project output is actually useful for them?
+  * How much time on an ongoing basis (ideally per-use) would they otherwise spend maintaining their home-rolled solution?
+
+When making these estimations, it's impossible to know with high precision _exactly_ how long any activities take.  That's not your goal.
```

Comment:
Nitpicky comment: I believe you are talking about _accuracy_, rather than _precision_ here.

Reply:
You're right - good point!